src/community/community.spec.ts

```diff
@@ -89,9 +89,27 @@ describe('Community', () => {
         throw new Error("error getting id")
       }
       await c.playTransactions(tree, [setDataTransaction("/somewhere/cool", "foo")])
-      await c.nextUpdate()
 
-      const respTip = await c.getTip(id)
+      const recursiveGetTip = ():Promise<CID> => {
```

Comment:
Do we expect this pattern to be needed for other clients? If so, should we just expose a method similar to this on the `Community` class itself - something like a `getTipWithRetry` or adding a retry option to `getTip`?

Reply:
I *expect* it to be but this is the only place in the code that does it so I'd like to wait for one more use.